src/common/parameters/Params.ts

```diff
@@ -0,0 +1,3 @@
+export type Params = {
+	[key: string]: string;
+};
```

Comment:
Da wir `asReference` und `editableBy` eigentlich immer dabei haben, könnte man überlegen, die hier auch zu hinterlegen (statt es komplett offen zu lassen).

```suggestion
export type Params = {
	asReference: string;
	editableBy: string;
	[key: string]: string;
};
```

Reply:
Das hatte ich überlegt und eine Version beinhaltete auch feste Werte. Die Parameter asReference und editableBy sind nur bei diesen GET Routen Standard. Andere Routen werden Parameter bekommen, die diese nicht beinhalten.

Daher habe ich es erstmal komplett frei gelassen.